language/jmv/examples/account/sources/Runtime.move

```diff
@@ -0,0 +1,12 @@
+/// Basic JMV Runtime support.
+module Jmv::Runtime {
+    /// Returns address of executing actor.
+    public native fun self(): address;
+
+    /// Returns a system time stamp, in nanos.
+    public native fun now(): u64;
```

Comment:
I don't think it's a good idea to expose `now`. This function call will be non deterministic and might cause state to diverage when retry. A heartbeat transaction to update the `now` as what we did in `DiemFramework` should be the way to go IMHO

Reply:
Sure, the way how time is represented should be aligned with what we need. However, I'm wondering whether a heartbeat would really help if it comes to retry. It would really need to be somehow a timestamp unique for the entire batch. Or is that what the heartbeat is?